src/PayloadParser.h

```diff
@@ -35,7 +35,7 @@ namespace snowcrash {
     };
 
     /** Request matching regex */
-    const char* const RequestRegex = "^[[:blank:]]*[Rr]equest" SYMBOL_IDENTIFIER "?" MEDIA_TYPE "?[[:blank:]]*";
+    const char* const RequestRegex = "^[[:blank:]]*[Rr]equest([[:blank:]]" SYMBOL_IDENTIFIER ")?" MEDIA_TYPE "?[[:blank:]]*";
```

Comment:
Allowing requests and responses doesn’t make sense because neither of those are plural (they are a single request or response). It would just make this more confusing.

Headers is a different case, because it can contain one or more headers. The task was to make these validations consistent, which now request and response matching are. Header's is consistent (disallows `headerz` etc), the one difference is that it supports plural "headers" since it handles one or multiple headers.